test/openzeppelin/ERC-20-extensions/ERC20Extensions.js

```diff
@@ -89,14 +89,12 @@ describe('@OZERC20Extensions Tests', function () {
       const newBalance = await ERC20Burnable.balanceOf(owner.address);
 
       // Check if the Transfer event was emitted to AddressZero
-      expect(burnReceipt.events[0].event).to.equal('Transfer');
-      expect(burnReceipt.events[0].args.to).to.equal(
-        ethers.constants.AddressZero
-      );
+      expect(burnReceipt.logs[0].fragment.name).to.equal('Transfer');
+      expect(burnReceipt.logs[0].args.to).to.equal(ethers.ZeroAddress);
 
       // Verify the new supply and new balance of the user
-      expect(newSupply).to.equal(initialSupply.sub(burnAmount));
-      expect(newBalance).to.equal(initialBalance.sub(burnAmount));
+      expect(newSupply).to.equal(initialSupply - BigInt(burnAmount));
```

Comment:
Might be more clear to declare `burnAmount` as `BigInt` (instead of `Number`) and avoid the conversion in each comparison.